sphinx4-core/src/main/java/edu/cmu/sphinx/frontend/feature/LiveCMN.java

```diff
@@ -167,8 +169,7 @@ public Data getData() throws DataProcessingException {
             while (initialList.size() < initialCmnWindow) {
                 input = getPredecessor().getData();
                 initialList.add(input);
-                if (input instanceof SpeechEndSignal
```

Comment:
Seems legit, but I didn't finish the review. I always forget that github
posts in line comments instantly.
On Mar 25, 2016 11:46 PM, "agur" <notifications@github.com> wrote:

> In sphinx4-core/src/main/java/edu/cmu/sphinx/frontend/feature/LiveCMN.java
> <https://github.com/cmusphinx/sphinx4/pull/52#discussion_r57510263>:
>
> > @@ -167,8 +169,7 @@ public Data getData() throws DataProcessingException {
> >              while (initialList.size() < initialCmnWindow) {
> >                  input = getPredecessor().getData();
> >                  initialList.add(input);
> > -                if (input instanceof SpeechEndSignal
>
> Should not matter will remove diff at that line for you next.
>
> What about the real change made, is that one looking good? Is there a
> better way we should consider for addressing those cases?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/cmusphinx/sphinx4/pull/52/files/872016fd7c278d70df85e099a1687f6982127d9d#r57510263>
>